SampleTest/MainPage.xaml

```diff
@@ -29,14 +29,14 @@
         <StackPanel Margin="16,48">
             <wgt:PeoplePicker x:Name="PeopleChooser"/>
             <TextBlock Margin="0,8,0,0" FontWeight="Bold">Picked People:</TextBlock>
-            <ItemsControl ItemsSource="{x:Bind PeopleChooser.PickedPeople}"
+            <!--<ItemsControl ItemsSource="{x:Bind PeopleChooser.PickedPeople}"
```

Comment:
Why is this commented out?

Reply:
I hadn't updated these yet to the new property name. I also think that exposes a WinUI bug now: https://github.com/microsoft/microsoft-ui-xaml/issues/1286 - need to investigate more.

Reply:
Ah, no that was trying to use the `TokenSpacing` property on the people picker causing that. I could uncomment this, but it causes some exceptions with types in the generated code unless I create a separate collection I think... I'll look